app/Http/Controllers/CutsController.php

```diff
@@ -53,11 +54,11 @@ public function show(Request $request, $instanceId, $cutId)
    */
   public function create(Request $request)
   {
-    $transfer = Transfer::findOrFail($request->transfer_id);
-    $instance = $transfer->preservationInstance;
+    $transfer = Transfer::findOrFail($request->transferId);
+    $instance = PreservationInstance::findOrFail($request->instanceId);
```

Comment:
it turns out that the form is populated specifically with the transfer's preservation_instance_id and can't be updated by the user. But this will help with anything terminal commands calling the controller directly!